src/App.tsx

```diff
@@ -15,14 +71,16 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this button should have `active` class only if all todos are completed */}
           <button
             type="button"
-            className="todoapp__toggle-all active"
+            className={classNames('todoapp__toggle-all', {
+              active:
+                currentTodos.length > 0 &&
+                currentTodos.every(todo => todo.completed),
```

Comment:
```suggestion
                currentTodos.some(todo =>! todo.completed),
```
it's more productive